### PR TITLE
Implements stub and debug profiles

### LINKED ISSUE #98: Add stub-run
Sometimes, it would be nice to check whether the plumbing of the different components of the pipeline is correctly done without needing to run even a small test data and with a container as small as possible.

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Implements a `test_stub` and `debug` profiles and example of how to use them would be:

```
nextflow run main.nf -profile test_stub,debug
```

Closes #98

### CODE REVIEW

File: modules/local/stimulus_split_csv.nf [hunk 1]
```diff
@@ -1,19 +1,25 @@
-
 process STIMULUS_SPLIT_CSV {
-
-    container 'alessiovignoli3/stimulus:latest'
+    
+    tag "$csv"
```
Comment (alessiovignoli) [inline modules/local/stimulus_split_csv.nf:3]:
```suggestion
    tag "$parsed_json
```

File: modules/local/stimulus_split_csv.nf [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
     output = "${csv.baseName}-${parsed_json.baseName}.csv"
     """
     launch_split_csv.py -c ${csv} -j ${parsed_json} -o ${output}
     """
+
+    stub:
+    output = "${csv.baseName}-${parsed_json.baseName}.csv"
+    """
+    touch ${output}
+    echo "launch_split_csv.py -c ${csv} -j ${parsed_json} -o ${output}" > launch_split_csv.sh
```
Comment (alessiovignoli) [inline modules/local/stimulus_split_csv.nf:23]: can you add a comment on why we do this line? for resume?

**Review by alessiovignoli — CHANGES_REQUESTED** [review]

File: modules/local/stimulus_split_csv.nf (hunk 2 above)
Comment (JoseEspinosa) [reply modules/local/stimulus_split_csv.nf:23]: No, this was actually a leak, I removed it

**Comment by mathysgrapotte** [comment]
@JoseEspinosa Can you merge from main after #108 is merged ? (or merge from there directly) HANDLE_TRAIN (and all similar names) have been changed to HANDLE_TUNE. This might cause merge conflicts.

**Comment by JoseEspinosa** [comment]
Sure let me know when #108 is merged.and will fix the merge conflicts here

**Review by alessiovignoli — APPROVED** [review]

**Comment by mathysgrapotte** [comment]
@JoseEspinosa #108 has been merged

**Pull request merged**